airbyte-integrations/connectors/destination-redis/src/main/resources/spec.json

```diff
@@ -8,7 +8,7 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Redis Destination Spec",
     "type": "object",
-    "required": ["host", "port", "username", "password", "cache_type"],
+    "required": ["host", "username", "port", "cache_type"],
```

Comment:
Just to confirm the reason `password` is no longer required is because of the new method `getOrGeneratePassword()`, right?